solver/src/solver/http_solver/model.rs

```diff
@@ -94,12 +94,12 @@ pub struct WeightedProductPoolParameters {
 }
 
 #[serde_as]
-#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct StablePoolParameters {
     #[serde_as(as = "HashMap<_, DecimalU256>")]
     pub reserves: HashMap<H160, U256>,
-    #[serde(with = "u256_decimal")]
-    pub amplification_parameter: U256,
+    #[serde(with = "ratio_as_decimal")]
+    pub amplification_parameter: BigRational,
```

Comment:
Should we pass numbers that are fractions as fractions to the solver instead of converting to decimal representation? For numbers like 1/3 there is no way to convert them to decimal in a lossless way. This is probably not the only place we could consider this.
Not important for this PR just an idea.

Reply:
Yea we should probably do this although I was trying to mimic what we are doing elsewhere for this PR.

Reply:
@vkgnosis I agree. That would already be possible with quasimodo, but would need to check if the MIP solver would handle it. Definitely a bigger change than this PR.